src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -133,7 +133,7 @@ public class SnowflakeSinkConnectorConfig {
   public static final String SNOWPIPE_STREAMING_ENABLE_SINGLE_BUFFER =
       "snowflake.streaming.enable.single.buffer";
 
-  public static final boolean SNOWPIPE_STREAMING_ENABLE_SINGLE_BUFFER_DEFAULT = true;
+  public static final boolean SNOWPIPE_STREAMING_ENABLE_SINGLE_BUFFER_DEFAULT = false;
```

Comment:
So we decided not to introduce / change default values of the SDK flush timeout / flush size?

Reply:
Not in 2.5.0. We'll try to figure this out for the next release.